staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -103,6 +119,7 @@ func TestApplyFlagValidation(t *testing.T) {
 
 	tests := []struct {
 		args        [][]string
+		enableAlpha string
```

Comment:
Maybe should be a `[]string`, but I also think it's reasonable to wait until we need to set more than one